webapi/src/main/scala/org/knora/webapi/http/ServerVersion.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2015 Lukas Rosenthaler, Benjamin Geer, Ivan Subotic,
+ * Tobias Schweizer, André Kilchenmann, and André Fatton.
+ * This file is part of Knora.
+ * Knora is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * Knora is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public
+ * License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.knora.webapi.http
+
+import akka.http.scaladsl.model.HttpResponse
+import akka.http.scaladsl.model.headers.Server
+import akka.http.scaladsl.server.Route
+import org.knora.webapi.BuildInfo
+import akka.http.scaladsl.server.Directives.respondWithHeader
+
+/**
+  * This trait can be used to add the [[Server]] header to an [[HttpResponse]].
+  */
+object ServerVersion {
```

Comment:
```suggestion
trait ServerVersion {
```

(or change the comment)

Reply:
ah, yes, thanks.